### ISSUE TITLE
Trashy output with rope-GPTQ-Extended context 

### ISSUE THREAD

**Issue by MSZ-MGS** [h2oai/h2ogpt#910]
I used below command line:
python generate.py --base_model=TheBloke/Llama-2-7b-Chat-GPTQ --load_gptq="gptq_model-4bit-128g" --use_safetensors=True --prompt_type=llama2 --hf_embedding_model=hkunlp/instructor-large --save_dir='save' --load_exllama=True --revision=gptq-4bit-32g-actorder_True --rope_scaling="{'alpha_value':4}" --score_model=None --do_sample=True --top_k_docs=20 --do_sample=True --temperature=0.7 --top_p=0.95 --repetition_penalty=1.1 --share=False --max_new_tokens=8192 --max_seq_len=8192 --max_time=1200

It works fine no issues what so ever, however, when I prompt (copy from a file and paste in prompt text input) a text that is 4714 Tokens it keeps generating trashy output, screenshot below:
![image](https://github.com/h2oai/h2ogpt/assets/65172063/6d6f38b7-59db-489d-b392-2cc2b759cb7c)

I noticed also a warning alert appeared in the console, screenshot below:
![image](https://github.com/h2oai/h2ogpt/assets/65172063/79ac250d-7555-4131-883d-64e1246f5410)

text file for reference.
[4714 Tokens.txt](https://github.com/h2oai/h2ogpt/files/12765252/4714.Tokens.txt)
Prompt: Make bullet points summary for below text:

**Comment by pseudotensor** [comment]
Yes, the RoPE stuff without fine-tuning is not very good.  Really a fine-tuned model is required unless one is simply trying to do naive retrieval.

The LM-Infinite approach seems much more promising in terms of not requiring fine-tuning.  There is no full implementation yet in open-source.
